docker-compose.two-backends.yaml

```diff
@@ -56,8 +56,11 @@ services:
       - DID_REGISTRY_ADDRESS=0xc15d5a57a8eb0e1dcbe5d88b8f9a82017e5cc4af
       - ENS_REGISTRY_ADDRESS=0xd7CeF70Ba7efc2035256d828d5287e2D285CD1ac
       - ENS_RESOLVER_ADDRESS=0xcf72f16Ab886776232bea2fcf3689761a0b74EfE
-      - IPFS_HOST=ewipfsgwtest.infura-ipfs.io
```

Comment:
The IPFS gateways don't work for did-auth-proxy?

Reply:
`ewipfsgwtest.infura-ipfs.io` hostname no longer exists, and I think it is better to use all variables in these example configs